distribution/client/src/lib/status.ts

```diff
@@ -1,20 +1,30 @@
 'use strict';
 
-const fs     = require('fs');
-const path   = require('path');
+import path from 'path';
+import fs from 'fs';
 
-const logger   = require('winston');
-const Storage  = require('./storage');
-const Checksum = require('./checksum');
+import * as logger from 'winston';
+import Storage from './storage';
+import Checksum from './checksum';
 
 
+export interface StatusOptions {
+  flavor?: string,
+};
```

Comment:
Seems like there could (should) be some something central for the flavors, as this code is the same as above. (just a comment, not a request for fix here)

Reply:
I agree, I think there's some refactoring we can do in the client to reduce some redundancies  :+1: